js/lykke2.js

```diff
@@ -0,0 +1,1171 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { NotSupported, ExchangeError, BadRequest, InsufficientFunds, InvalidOrder, DuplicateOrderId } = require ('./base/errors');
+const Precise = require ('./base/Precise');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class lykke2 extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'lykke2',
+            'name': 'Lykke',
+            'countries': [ 'UK' ],
+            'version': '2',
+            // 300 requests per minute per method => 60000ms / 300 = 200 (/api/orders/*)
+            // 120 requests per minute per method => ( 60000ms / rateLimit ) / 120 = cost = 2.5 (/api/*)
+            'rateLimit': 200, // TODO: optimize https://lykkecity.github.io/Trading-API/#request-rate-limits
+            'has': {
+                'CORS': undefined,
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'cancelAllOrders': true,
+                'cancelOrder': true,
+                'createOrder': true,
+                'editOrder': false,
+                'fetchBalance': true,
+                'fetchBorrowRate': false,
+                'fetchBorrowRateHistories': false,
+                'fetchBorrowRateHistory': false,
+                'fetchBorrowRates': false,
+                'fetchClosedOrders': true,
+                'fetchCurrencies': true,
+                'fetchDepositAddress': true,
+                'fetchDeposits': false,
+                'fetchFundingFees': false,
+                'fetchFundingHistory': false,
+                'fetchFundingRate': false,
+                'fetchFundingRateHistory': false,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': false,
+                'fetchMarkets': true,
+                'fetchMarkOHLCV': false,
+                'fetchMyTrades': true,
+                'fetchOHLCV': 'emulated',
```

Comment:
this method does not exist in this code, should we change it to false?

Reply:
It turns out `emulated` is default option in Exchange.js, called and build OHLCV with `fetchTrades`, should I keep this?

Reply:
It's ok if we keep `emulated` there for now.